net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checkstyle-metadata_1_0.dtd

```diff
@@ -8,138 +8,138 @@
 -->
```

Comment:
@Calixte What is the purpose of https://github.com/checkstyle/eclipse-cs/tree/master/docs and how is it updated?

I noticed it has the same DTD files at https://github.com/checkstyle/eclipse-cs/tree/master/docs/dtds that I updated in this PR, but nothing required those files to be updated as well.

Reply:
The `/docs` directory is the directory configured to be served by GitHub Pages at https://checkstyle.org/eclipse-cs/
It is automatically generated during the release when running `mvn package` on `net.sf.eclipsecs.doc`.
I believe the two are decoupled so the source doc `net.sf.eclipsecs.doc` can be used to generate the website, but also the Eclipse Help content.
See those lines in the release script: https://github.com/checkstyle/eclipse-cs/blob/master/release.sh#L70-L75

Reply:
So basically `/docs` is the suppose to be the last release's documentation (10.4.0) as we are in snapshot for the next release. Eclipse-CS doesn't use another repo for the hosted site like main repo does at https://github.com/checkstyle/checkstyle.github.io ?

So basically https://checkstyle.org/eclipse-cs/ is a mirror of http://checkstyle.github.io/eclipse-cs/ which points to https://github.com/checkstyle/eclipse-cs/tree/master/docs/ in virtually real time?

Reply:
You are correct.

Reply:
Thanks @Calixte

Reply:
> nothing required those files to be updated as well

Actually, those fail when I build now.